test/extensions/clusters/original_dst/original_dst_cluster_test.cc

```diff
@@ -86,7 +85,9 @@ class OriginalDstClusterTest : public Event::TestUsingSimulatedTime, public test
         });
     cluster_->initialize([&]() -> void { initialized_.ready(); });
   }
-
+  // NOTE: the cm_.trafficStatNames() is referenced by the MockClusterInfo::trafficStats(), it
+  // should out live the MockClusterInfo object.
```

Comment:
If we have to comment all of these kinds of dependencies it will get really verbose :)

To me it seems obvious that since cm_ is passed into factory_context by ref it should outlive it.

So I don't think this comment is necessary, but I'm OK if you prefer to leave it in.

Reply:
removed.